wifiphisher/common/phishinghttp.py

```diff
@@ -19,13 +20,24 @@ def get(self, url):
         self.redirect("http://10.0.0.1:8080/")
 
 
-class ShowPhishingPageHandler(tornado.web.RequestHandler):
+class CaptivePortalHandler(tornado.web.RequestHandler):
 
     def get(self, url):
+        client_request = self.request.path[1:]
         try:
```

Comment:
`Try` is definitely required here as many "weird requests" will throw exceptions. Catching all exceptions at once is not a good practice though. An improvement would be to catch them separately.